posts/2017-10-02-serverless-vs-containers.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Serverless (FaaS) vs. Containers - when to pick which?
+description: Docker, Kubernetes, Serverless? Let's discuss the ups and downs of each. 
+date: 2017-10-02
+layout: Post
+thumbnail: https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/serverless-vs-containers.png
+authors:
+  - PhilippMuns
+---
+
+Contrary to popular thought, Functions-as-a-Service (FaaS) and Containers have some pretty important things in common.
```

Comment:
FaaS are containers in most cases. I guess it's more about FaaS vs Container Orchestrators?